aten/src/ATen/native/native_functions.yaml

```diff
@@ -3101,6 +3101,10 @@
   dispatch:
     CompositeExplicitAutograd: mkldnn_convolution
 
+- func: mkldnn_convolution_transpose(Tensor self, Tensor weight, Tensor? bias, int[] padding, int[] output_padding, int[] stride, int[] dilation, int groups) -> Tensor
+  dispatch:
+    CompositeExplicitAutograd: mkldnn_convolution_transpose
```

Comment:
Now that we have a single `convolution` and `convolution_backward` op, maybe these mkldnn variants should be folded directly into `convolution`? (cc @albanD, I'm not super familiar with how the new implementation works)

Then again it looks like we already have an existing `mkldnn_convolution` op that wasn't folded into `convolution` yet.

Reply:
Yes they should be folded in there and shouldn't have an entry in native_functions.yaml anymore and the general convolution will call into that directly.

Reply:
at the current moment, `mkldnn_convolution_transpose` is treated in the same way as other `mkldnn_` functions and `cudnn`/`miopen` counterpart, feel free if additional changes are required.

Reply:
this entry has removed and used stub instead.